docs/GraFx-Developers/connectors/build-media-connector/index.md

```diff
@@ -0,0 +1,284 @@
+# Guide to building your own Media Connector
+
+## Pre-requisites
+
+See also [pre-requisites](../pre-requisites/)
+
+- Have 'npm' installed
+- Have 'nodejs' installed
+- Knowledge of nodejs
+- IDE
+	- [Visual Studio Code](https://code.visualstudio.com/)
+	- [Rider](https://www.jetbrains.com/rider/)
+	
+## Local install location
+
+Before you start, choose where you want to have the local setup. This directory will hold all the files (.js) for your connector.
+
+Go to (or make) directory, where you want to start making your connector  
+e.g. "/Users/<yourname>/mediaconnector-mydam"
+
+## Install the CLI
+
+The CLI (Command Line Interface) is part of the tooling we offer to get you up and running.
+
+The CLI Tool has several commands that will perform tasks like building, deploying, getting the latest files, etc.
+
+Install the command line tools. This will help you to install, build and deploy your Media connector.
+
+The "-g" option will make sure the command line tool is available on a "global" level, meaning also outside of your directory. In case you want to work on several connectors, you can call the tool from any directory.
+
+``` bash
+npm install -g @chili-publish/connector-cli
+```
+
+You can now execute the command (in the terminal, in the right directory)
+
+To understand what the CLI can do, you can now execute
+This will give you an overview of the available commands
+
+``` csh
+npx connector-cli
+```
+
+
+Initialize: Means building the Boiler Plate. This is the basic code you'll need to get started.
+
+``` csh
+npx connector-cli init
+```
+
+You now have the basic code. In that code, we have defined some dependencies. To download these, execute the install command.
+
+
+``` csh
+npm install
+```
+
+The result should look something like this
+
+```csh
+added 112 packages, and audited 113 packages in 5s
+
+35 packages are looking for funding
+  run `npm fund` for details
+
+found 0 vulnerabilities
+
+```
+
+Now it's time to create your custom code.
+
+This happens in the file "connector.ts". This is a typescript files.
+
+By default, there will be no implementations. Your IDE can do this for you, to build the default interface methods.
```

Comment:
In context of https://github.com/chili-publish/studio-connector-contrib/pull/16, we've added `default` implementation, so you might skip the next step